QueueTest/QueueTest.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+	  <OutDir>..\bin</OutDir>
+

```

Comment:
`<GenerateDocumentationFile>True</GenerateDocumentationFile>`
You haven't turned on this option